src/live_restore/live_restore_private.h

```diff
@@ -38,6 +38,7 @@ struct __wti_live_restore_hole_node {
 struct __wti_live_restore_file_handle {
     WT_FILE_HANDLE iface;
     WT_FILE_HANDLE *source;
+    size_t source_size;
```

Comment:
Where is this field set?

Reply:
Oops. I missed initializing it. Good catch. Fixed.